tests/fixtures/commands/component-check.yml

```diff
@@ -307,6 +336,8 @@
           drupal_patches: false
   tokens: ''
   resources:
+    - from: 'sample-docker-default.yml'
```

Comment:
Do we need to always provide this file ?

Reply:
Apparently, we don't need it. It must have been added when functionality was not fully ready. Good catch, thank you. It's been fixed.